applications/tari_base_node/src/grpc/base_node_grpc_server.rs

```diff
@@ -806,7 +806,7 @@ impl tari_rpc::base_node_server::BaseNode for BaseNodeGrpcServer {
         let peers = self
             .comms
             .peer_manager()
-            .all()
+            .discovery_syncing()
```

Comment:
This isn't where peer sync happens :) 

You need to change the peer to peer RPC. 

`comms/dht/src/rpc/service.rs`